bip-0495.mediawiki

```diff
@@ -0,0 +1,93 @@
+* Maintain user sovereignty — no enforcement, only voluntary participation.
+* Facilitate historical auditing and policy planning by future communities.
+
+==Rationale==
+Bitcoin is a finite resource. As time passes, more of it becomes irretrievably lost. While this contributes to scarcity and value, it also risks rendering Bitcoin illiquid in the far future.
+
+This proposal '''does not suggest forcibly reclaiming coins''', but rather introduces a '''social signal system''' to help distinguish between active and dormant wallets over extreme time horizons (e.g., 100+ years). This gives future generations the opportunity to evolve responsibly, should they choose to discuss reactivation of provably abandoned funds.
+
+==Specification==
+
+===1. Long-Term Activity Proof (LTAP)===
+* A message signed with a wallet’s private key proving it is still accessible.
+* Suggested to be generated once every 10 years, optionally published to a decentralized registry (on-chain, Layer-2, or other).
+* Format of the message:
+
+<pre>
+LTAP - &lt;Wallet Address&gt; - &lt;UTC Timestamp&gt; - &lt;Optional Metadata&gt;
+</pre>
+
+Example:
+
+<pre>
+LTAP - bc1p8j0ckwf372usvlvve0aesen7jzplxf2fgysg3fur3lt6nlh8r9wq6sww6p - 2125-01-01T00:00:00Z - still alive
+</pre>
+
+* The signature must be verifiable with standard Bitcoin message signing tools.
+* Publication could be via any public timestamped medium (blockchain, IPFS, etc.)
+
+===2. Dormancy Threshold===
+* A wallet that shows '''no LTAP for more than 100 years''' may be '''socially''' categorized as dormant.
+* No forced action is defined by this BIP.
+* Communities may discuss use cases such as statistical tracking or theoretical economic models.
+
+===3. Privacy Considerations===
+* No identity or off-chain data is required.
+* LTAPs are purely cryptographic and voluntary.
+* No public registry is mandatory — only suggested for future reference.
+
+==Compatibility==
+* 100% backward-compatible with Bitcoin.
+* Does not change consensus rules.
+* LTAP system operates entirely '''off-chain''' unless voluntarily recorded on-chain.
+
+==Reference Implementation==
+A sample LTAP generator may look like:
+
+<pre>
+bitcoin-cli signmessage "&lt;wallet_address&gt;" "LTAP - &lt;wallet_address&gt; - &lt;timestamp&gt; - optional note"
+</pre>
+
+==A note on the importance of this proposal==
+This update gives future generations the opportunity to own and mine more Bitcoin, rather than constantly losing it, which ultimately leads to a depleted supply of Bitcoin. This is considered unfair to future generations, especially as the total Bitcoin mining capacity is depleted.
+
+==Copyright==
+This work is licensed under the BSD 2-Clause License.
+
+== Supporting Reference ==
+
+For full context, motivations, and background behind this BIP, please refer to the external repository:
+https://github.com/Danger-MaN/Bitcoin_bip-bdrp
+And My Twitter X:
+https://X.CoM/Page_Magdi
```

Comment:
I don't see additional information in the linked external repository.

I don't see other BIP authors providing their X/twitter social media account and these can be volatile links.